students/synowiec_krzysztof/lesson_02_flow_control/chocolate_bar.py

```diff
@@ -0,0 +1,13 @@
+def main():
+    m = int(input())
+    n = int(input())
+    k = int(input())
+    max = m*n
```

Comment:
variable name `max` shadows python built-in function `max()`
You can check those at https://docs.python.org/3/library/functions.html